libraries/libpensol/pom.xml

```diff
@@ -118,6 +118,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>pentaho-kettle</groupId>
```

Comment:
@YuryBY again, bringing in kettle-core as a dependency for libpensol is to be avoided